src/yacam.cpp

```diff
@@ -1040,6 +966,77 @@ RC YACAM::errFlLn( const char *s, ...)	// error message "%s in <mWhat> <mPathNam
 			   (char *)MH_I0118,		// "%s '%s' (near line %d):\n  %s"
 			   mWhat,  mPathName ? mPathName : "bug",  mLineNo,  buf );
 }
+#if 0 // Methonds not used, but were modified to fit c-style io, not tested. sha d06014d81f75e5ec2b9d4e31750dd0c076afae46
```

Comment:
1. Spell check
2. Is "c-style io" the right term?
3. Do we need the sha here? Generally we only need a sha if we're deleting something.

Reply:
I think we settled on not putting SHAs in comments because version control systems are not forever.

Reply:
That's been Chip's argument for a while. I thought that if we did something like this, we'd just mention that last release version where the code appeared.